docs/design/podService.md

```diff
@@ -0,0 +1,24 @@
+# Add support for TLS certificates for CSI Addons communication
+
+The Kubernetes CSI Addons project currently connects the CSI Addons Manager to the add-on sidecar without TLS in place.
+We propose a introduction of an arugment which enabled should mount the TLS certificates into the sidecar container.
```

Comment:
Please provide a code snippet that describes the desired outcome of this new field within the API you would like to augment. 

The code snippet should be descriptive enough so the following will be clear:
1. The name of the new field
2. The type of the new field 
3. The API is being introduced into
4. The location (path) in the API where the new field will be added
5. An explanation of what the field is describing
6. And validation rules needed on the field (can be provided as standalone text or as a comment within the snippet)